src/Functions/castOrDefault.cpp

```diff
@@ -376,66 +376,475 @@ SELECT accurateCastOrDefault('abc', 'UInt32')
+    FunctionDocumentation toUInt16OrDefault_documentation = {toUInt16OrDefault_description, toUInt16OrDefault_syntax, toUInt16OrDefault_arguments, toUInt16OrDefault_returned_value, toUInt16OrDefault_examples, toUInt16OrDefault_introduced_in, toUInt16OrDefault_category};
+
     factory.registerFunction("toUInt16OrDefault", [](ContextPtr context)
-        { return std::make_shared<FunctionCastOrDefaultTyped>(context, "toUInt16OrDefault", std::make_shared<DataTypeUInt16>()); });
+        { return std::make_shared<FunctionCastOrDefaultTyped>(context, "toUInt16OrDefault", std::make_shared<DataTypeUInt16>()); },
+        toUInt16OrDefault_documentation);
+    FunctionDocumentation::Description toUInt32OrDefault_description = R"(
+Like [`toUInt32`](#touint32), this function converts an input value to a value of type [UInt32](../data-types/int-uint.md) but returns the default value in case of an error.
+If no `default` value is passed then `0` is returned in case of an error.
+    )";
+    FunctionDocumentation::Syntax toUInt32OrDefault_syntax = "toUInt32OrDefault(expr[, default])";
+    FunctionDocumentation::Arguments toUInt32OrDefault_arguments = {
+        {"expr", "Expression returning a number or a string representation of a number.", {"String", "Int", "Float"}},
+        {"default", "Optional. The default value to return if parsing is unsuccessful.", {"UInt32"}}
+    };
+    FunctionDocumentation::ReturnedValue toUInt32OrDefault_returned_value = {"Value of type UInt32 if successful, otherwise returns the default value if passed or 0 if not.", {"UInt32"}};
+    FunctionDocumentation::Examples toUInt32OrDefault_examples = {
+        {"Successful conversion", "SELECT toUInt32OrDefault('32', CAST('0', 'UInt32'))", "32"},
+        {"Failed conversion", "SELECT toUInt32OrDefault('abc', CAST('0', 'UInt32'))", "0"}
+    };
+    FunctionDocumentation::Category toUInt32OrDefault_category = FunctionDocumentation::Category::TypeConversion;
+    FunctionDocumentation::IntroducedIn toUInt32OrDefault_introduced_in = {21, 11};
+    FunctionDocumentation toUInt32OrDefault_documentation = {toUInt32OrDefault_description, toUInt32OrDefault_syntax, toUInt32OrDefault_arguments, toUInt32OrDefault_returned_value, toUInt32OrDefault_examples, toUInt32OrDefault_introduced_in, toUInt32OrDefault_category};
+
     factory.registerFunction("toUInt32OrDefault", [](ContextPtr context)
-        { return std::make_shared<FunctionCastOrDefaultTyped>(context, "toUInt32OrDefault", std::make_shared<DataTypeUInt32>()); });
+        { return std::make_shared<FunctionCastOrDefaultTyped>(context, "toUInt32OrDefault", std::make_shared<DataTypeUInt32>()); },
+        toUInt32OrDefault_documentation);
+    FunctionDocumentation::Description toUInt64OrDefault_description = R"(
+Like [`toUInt64`](#touint64), this function converts an input value to a value of type [UInt64](../data-types/int-uint.md) but returns the default value in case of an error.
```

Comment:
```suggestion
Like [`toUInt64`](#toUInt64), this function converts an input value to a value of type [UInt64](../data-types/int-uint.md) but returns the default value in case of an error.
```